packages/now-routing-utils/test/superstatic.spec.js

```diff
@@ -386,6 +387,11 @@ test('convertRewrites', () => {
       dest: '/api/user?id=$1',
       check: true,
     },
+    {
+      src: '^(?:\\/([^\\/]+?))$',
```

Comment:
Isn't this supposed to have an optional trailing slash?

Reply:
Not that I'm aware of

Reply:
That's what Next does

Reply:
It's strict: false or something

Reply:
cc @styfle

Reply:
We use `{ strict: true }` in Next.js for custom routes. [Related line](https://github.com/zeit/next.js/blob/7038d275d024e81961c0563f68d0aba1ede78ed2/packages/next/next-server/server/lib/path-match.ts#L11)

Reply:
Oh yeah. I was thinking of:
```
{
"check": true,
"dest": "/[slug]",
"src": "^/([^/]+?)(?:/)?$"
},
```

We should probably make it consistently false because static files match with or without a trailing slash